htdocs/sendtoken.php

```diff
@@ -162,14 +182,42 @@
     } else {
         $token = session_id();
     }
+    session_write_close();
+}
+
 
+#==============================================================================
+# Check tokenform
+#==============================================================================
+
+if ( !$result ) {
+    $formtoken = strval($_REQUEST["formtoken"]);
+    ini_set("session.use_cookies",0);
+    ini_set("session.use_only_cookies",1);
+    ini_set("session.use_strict_mode",0);
+    session_name("formtoken");
+    session_id($formtoken);
+    session_start();
+    if( $_SESSION['formtoken'] == $formtoken )
+    {
+        # Remove session
+        session_unset();
+        session_destroy();
+    }
+    else
+    {
+        error_log("Invalid form token: sent: $formtoken, stored: " . $_SESSION['formtoken']);
+        $result = "invalidformtoken";
+    }
+    session_write_close();
 }
 
 #==============================================================================
 # Send token by mail
 #==============================================================================
 if ( !$result ) {
 
+
```

Comment:
Was it intentional to add this blank line?

Reply:
No. I'll remove it